Home-page.html

```diff
@@ -25,7 +25,7 @@ <h1 class="text-center">Gang Out</h1>
       <div class="card" id="userdata" style="width: 18rem;">
         <img src="..." class="card-img-top" alt="...">
         <div class="card-body">
-          <p class="card-text">Franky Two Toes</p>
+          <p class="card-text">...</p>
```

Comment:
do we want to to add text here or will that come in a later PR?

Reply:
Yeah that will be added later